core/tests/puppeteer-acceptance-tests/specs/curriculum-admin/create-publish-unpublish-and-delete-topic-and-skill.spec.ts

```
@@ -51,59 +52,84 @@ describe('Curriculum Admin', function () {
+      'test-subtopic-one',
+      'Test Topic 1'
+    );
+
+    // Create skill, and add it to diagnostic test.
+    await curriculumAdmin.createSkillForTopic(
+      'Test Skill 1',
+      'Test Topic 1',
+      false
+    );
+    await curriculumAdmin.createQuestionsForSkill('Test Skill 1', 3);
+    await curriculumAdmin.assignSkillToSubtopicInTopicEditor(
+      'Test Skill 1',
+      'Test Subtopic 1',
+      'Test Topic 1'
+    );
+    await curriculumAdmin.addSkillToDiagnosticTest(
+      'Test Skill 1',
+      'Test Topic 1'
+    );
+
+    // Publish topic.
+    await curriculumAdmin.publishDraftTopic('Test Topic 1');
+    await curriculumAdmin.expectToBeInTopicAndSkillsDashboardPage();
+    await curriculumAdmin.expectTopicToBePublishedInTopicsAndSkillsDashboard(
+      'Test Topic 1',
+      0, // Story added.
+      1, // Subtopic added.
+      1 // Skill added.
+    );
+
+    await curriculumAdmin.openTopicEditor('Test Topic 1');
+    await curriculumAdmin.expectUnpublishTopicButtonToBeVisible();
+  });
+
+  it('should be able to create a skill', async function () {
+    await curriculumAdmin.navigateToTopicAndSkillsDashboardPage();
+    await curriculumAdmin.navigateToSkillsTab();
+
+    await curriculumAdmin.clickOnCreateNewSkillButtonInSkillDashboard();
+    await curriculumAdmin.fillSkillDetailsInNewSkillModal(
+      'Test Skill 2',
+      'Test Review Material'
+    );
+    const newPage = await curriculumAdmin.clickOnElementAndGetNewPage('Save');
+    await curriculumAdmin.expectToBeInSkillEditorPage(newPage);
+  });
+
+  it('should be able to unpublish a topic', async function () {
+    await curriculumAdmin.unpublishTopic('Test Topic 1');
+    await loggedInUser.expectTopicLinkReturns404('test-topic-one');
+  });
+
+  it('should be able to delete a topic', async function () {
+    await curriculumAdmin.deleteTopic('Test Topic 1');
+    await curriculumAdmin.expectTopicNotInTopicsAndSkillDashboard(
+      'Test Topic 1'
+    );
+  });
```

There is another expectation after deleting a topic: `The respective links for Topic should return a 404 error`. If this is not correct, update the CUJ doc.

---

I am assuming you are talking about last row "delete a topic". Again I deleted it in suggestion, but was rejected. I am deleting it directly.